### ISSUE TITLE
Need to modify rubric in  02.22.txt

### ISSUE THREAD

**Issue by GoogleCodeExporter** [DivinumOfficium/divinum-officium#258]
```
In file 02-22.txt,

(sed rubrica 1960 haec versus omittuntur) is bad Latin. It should be
(sed rubrica 1960 hi versus omittuntur). However, this modification is not 
handled correctly by the program.
```

Original issue reported on code.google.com by `roberto....@gmail.com` on 19 Nov 2014 at 6:37

**Comment by gregordick** [comment]
Also 'hic versus' (and not 'hoc'). I seem to have thought that 'versus' was neuter.

**Comment by APMarcello3** [comment]
Closed by 3b74e23ff.
